data/ecosystems/c/celo.toml

```diff
@@ -13,7 +13,10 @@ sub_ecosystems = [
   "Cask",
   "Celo Tracker",
   "Celostrials",
+  "Centrifuge",
```

Comment:
Please remove this one, we are keeping it "Secondary" to Celo already so no need to add it here